controllers/admin/AdminLinkWidgetController.php

```diff
@@ -218,11 +219,45 @@ protected function buildHelper()
         return $helper;
     }
 
-    public function validateForm($data)
+    /**
+     * @param array $fields
+     * @return bool
+     */
+    public function validateLinkName(array $fields)
```

Comment:
You're technically right, both ways display a message saying you cannot leave this input blank.

But I would prefer the new message:
![capture du 2016-08-23 15-47-40](https://cloud.githubusercontent.com/assets/6768917/17894259/0a5ee85a-6949-11e6-9ded-ccf5a2ae0938.png)


Instead of the aggressive one :wink: :
![capture du 2016-08-23 15-47-11](https://cloud.githubusercontent.com/assets/6768917/17894260/116eb9e0-6949-11e6-9c76-9e4604f74c0a.png)